specs/_features/focil/validator.md

```diff
@@ -0,0 +1,116 @@
+
+## New inclusion list committee assignment
+
+A validator may be a member of the new Inclusion List Committee (ILC) for a given slot.  To check for ILC assignments the validator uses the helper `get_ilc_assignment(state, epoch, validator_index)` where `epoch <= next_epoch`.
+
+ILC selection is only stable within the context of the current and next epoch.
+
+```python
+def get_ilc_assignment(
+        state: BeaconState,
+        epoch: Epoch,
+        validator_index: ValidatorIndex) -> Optional[Slot]:
+    """
+    Returns the slot during the requested epoch in which the validator with index `validator_index`
+    is a member of the ILC. Returns None if no assignment is found. 
+    """
+    next_epoch = Epoch(get_current_epoch(state) + 1)
+    assert epoch <= next_epoch
+
+    start_slot = compute_start_slot_at_epoch(epoch)
+    for slot in range(start_slot, start_slot + SLOTS_PER_EPOCH):
+        if validator_index in get_inclusion_list_committee(state, Slot(slot)):
+            return Slot(slot)
+    return None
+```
+
+### Lookahead
+
+`get_ilc_assignment` should be called at the start of each epoch to get the assignment for the next epoch (`current_epoch + 1`). A validator should plan for future assignments by noting their assigned ILC slot. 
+
+## New proposer duty
+
+### Block proposal
+
+Proposers are still expected to propose `SignedBeaconBlock` at the beginning of any slot during which `is_proposer(state, validator_index)` returns `true`. The mechanism to prepare this beacon block and related sidecars differs from previous forks as follows:
+
+#### Update execution client with inclusion lists
+
+The proposer should call `engine_updateInclusionListV1` at `PROPOSER_INCLUSION_LIST_CUT_OFF` into the slot with the list of the inclusion lists that gathered since `LOCAL_INCLUSION_LIST_CUT_OFF`
+
+
+## New inclusion list committee duty
+
+Some validators are selected to submit local inclusion list. Validators should call `get_ilc_assignment` at the beginning of an epoch to be prepared to submit their local inclusion list during the next epoch. 
+
+A validator should create and broadcast the `signed_inclusion_list` to the global `inclusion_list` subnet by the `LOCAL_INCLUSION_LIST_CUT_OFF` in the slot, unless a block for the current slot has been processed and is the head of the chain and broadcast to the network.
+
+#### Constructing a local inclusion list
+
+The validator creates the `signed_local_inclusion_list` as follows:
+- First, the validator creates the `local_inclusion_list`.
+- Set `local_inclusion_list.slot` to the assigned slot returned by `get_ilc_assignment`.
+- Set `local_inclusion_list.validator_index` to the validator's index.
+- Set `local_inclusion_list.parent_hash` to the block hash of the fork choice head.
+- Set `local_inclusion_list.parent_root` to the block hash of the fork choice head.
```

Comment:
What's the difference between these two?

Reply:
one is consensus layer construct, one is execution layer construct, fixing the sentences to clarify